hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -119,14 +119,13 @@
     }
     // Sim mode that enables build randomization. See the `build_seed` mode
     // defined in `hw/dv/tools/dvsim/common_modes.hjson` for more details.
-    // TODO: enable the following pre_build_cmds once the otp init error is fixed
-    // "cd {proj_root} && ./util/design/gen-otp-mmap.py --seed {seed}"
     {
       name: build_seed
       pre_build_cmds: [
         '''cd {proj_root} && ./util/topgen.py -t {ral_spec} \
-               -o hw/top_earlgrey --rnd_cnst_seed {seed}
-        '''
+               -o hw/top_earlgrey --rnd_cnst_seed {build_seed}
+        ''',
+        "cd {proj_root} && ./util/design/gen-otp-mmap.py --seed {build_seed}"
```

Comment:
we should also add lifecycle state generation in a follow up in this place.

Reply:
sounds good, thanks Michael, i am going to do that in a following PR!